cogs/birthday/birthday.py

```diff
@@ -543,39 +543,56 @@ async def setSelfBirthday(self, ctx: Context, *, birthday: MonthDayConverter):
         birthdayStr = "{0:%B} {0:%d}".format(birthday)
 
         if birthdayConfig:
-            confirmationStr = (
-                f"Are you sure you want to set your birthday to "
-                f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}? "
-                "Only administrators and moderators can reset your birthday afterwards. "
-                f"Type {bold('`yes`', escape_formatting=False)} to confirm."
+            confirmationStr = "\n".join(
+                (
+                    f"Are you sure you want to set your birthday to "
+                    f"{spoiler(bold(birthdayStr, escape_formatting=False), escape_formatting=False)}? "
+                    "Only administrators and moderators can reset your birthday afterwards.",
+                    f"Type {bold('`yes`', escape_formatting=False)} to confirm.",
+                )
             )
 
-            try:
+            async def mainFlow(destination, carefree: bool = False):
```

Comment:
Add type hint [`Union`](https://docs.python.org/3/library/typing.html#typing.Union) for `destination` here. We expect either [`Context`](https://discordpy.readthedocs.io/en/v1.7.3/ext/commands/api.html#discord.ext.commands.Context) or [`User`](https://discordpy.readthedocs.io/en/v1.7.3/api.html#discord.User).
```suggestion
            async def mainFlow(destination: Union[Context, User], carefree: bool = False):
```

Reply:
Thinking back on this I don't know why I sent context or user; we have no need for either one of these since we are just trying to extract the channel out of it. Instead the channel should be used directly as an argument and we can just check for that instead of expecting a Context or User.

Reply:
Yeah if we don't need anything from `Context` or `User` we can just drop this argument.

Reply:
But wouldn't the checks require an identity check? We common do post-prompt check to ensure answer to the prompt is in the same channel as the channel in which the prompt was activated, and the answerer is the same as the prompt activator.

Reply:
Hmm well I guess we can have that from the `Context` object in the outer scope enclosing `mainFlow`.